### ISSUE TITLE
HTML proofer build test is failing

### ISSUE THREAD

**Issue by DaveSkender** [GSA/digitalcorps.gsa.gov#16]
## Describe the bug

When replacing `a11y` with `pa11y` (#14) the `npm run test` command was shortened to just do the `html-proofer` parts.  Run alone, it uncovered a pre-existing run failure that was trying to verify internal navigation links with the full live domain `site.url` prefix, which 1) doesn't exist before initial deployment, and 2) shouldn't be happening for internal links -- checking the production URL for a `build` site may produce false negatives (it's not testing the right site).

```bash
- _site/opportunity/index.html
  *  External link https://digitalcorps.gsa.gov/opportunity/ failed: response code 0 means something's wrong.
             It's possible libcurl couldn't connect to the server or perhaps the request timed out.
             Sometimes, making too many requests at once also breaks things.
             Either way, the return message (if any) from the server is: Couldn't resolve host name
```

## To Reproduce

1. Uncomment the `npm run test-links` section of the Circle CI `.circleci/config.yml` file
2. Run `npm run test-links`

## Expected behavior

Should not produce an error for pages that exist on the locally build site and do not exist on the live site.

## Workaround

Do not run HTML URL proofing
